bottlerocket/types/src/agent_config.rs

```diff
@@ -495,12 +495,25 @@ fn k8s_version_valid() {
 pub struct WorkloadTest {
     pub name: String,
     pub image: String,
+    #[serde(default)]
+    pub gpu: bool,
 }
 
 #[derive(Debug, Clone, Default, Serialize, Deserialize, Configuration, Builder)]
 #[serde(rename_all = "camelCase")]
 #[crd("Test")]
 pub struct WorkloadConfig {
-    pub kubeconfig_base64: String,
-    pub plugins: Vec<WorkloadTest>,
+    pub kubeconfig_base64: Option<String>,
```

Comment:
Is there a reason this was changed to an option?

Reply:
I'm trying to reuse this struct for ECS testing. With K8s, this is needed for Sonobuoy to have the kubeconfig to speak to the cluster. With ECS there is no such thing. So I switched it to `Option` to allow either case.

Reply:
So the `EcsWorkloadTestConfig` is not going to be used?

Reply:
Oops! Now that I actually look again, I think I was originally going down the route of trying to use `WorkloadConfig` for both types of tests, then realized I needed other things that required the `EcsWorkloadTestConfig` struct. I'll change this back.